lib/commons/dom/find-nearby-elms.js

```diff
@@ -37,3 +45,13 @@ function loopGridCells(gridCells, boundaries, cb) {
     }
   }
 }
+
+const hasFixedPosition = memoize(vNode => {
+  if (!vNode) {
+    return false;
+  }
+  if (vNode.getComputedStylePropertyValue('position') === 'fixed') {
```

Comment:
We should also consider `sticky` position 

```suggestion
  if (['fixed', 'sticky'].includes(vNode.getComputedStylePropertyValue('position')) {
```

Reply:
I don't think this works for sticky elements. This is already a fairly naive approach to fixed elements... It's sort of just hoping that fixed elements are never in a subgrid, and that within a fixed position element there isn't a scrollable region.

But for sticky this is not the way to go. Sticky element's aren't like fixed position elements. They are sticky to their container, not fixed to the viewport. Sticky elements are much more like position:relative then they are like position:fixed. While not optimal, I think treating them like relative elements is better than treating them like sticky elements.

Ideally the solve for sticky would be for the grid to treat them as though they are on their original position, rather than take their current bounding box as the correct placement for them. That's out of scope for this PR, and for the 4.5 release I think too. I think we need to revisit the grid for 4.6 to address these problems, but we're too late at this stage.

Reply:
> Ideally the solve for sticky would be for the grid to treat them as though they are on their original position, rather than take their current bounding box as the correct placement for them

I don't think that's possible. The grid positions the elements where they are at the time it is created. It has no way of telling if the elements current position is not its original position before scrolling occurred. Once the sticky elements scroll boundary is passed, a sticky element is no different than a fixed element in terms of the bounding box and position. The problem is there's no way to tell where the elements scroll boundary is unless the grid were to take scrolling into account, which it doesn't as it treats the entire page as one long document.

I don't mind saying we don't want to add code to deal with sticky elements into this fix, so long as we understand that the fix for sticky elements is probably not going to be in the grid itself.